### DISCUSSION TITLE
Added the workflow but its not appearing, meanwhile upgrading workflow works

### DISCUSSION THREAD

**Discussion by zaherg** [community/community#26118] [Actions]
<p>Hey,</p>
<p>Am not sure what is the issue, but adding a workflow for a repo does not work unless it’s a repo with old action. For example, the following <a href="https://github.com/linuxjuggler/redesigned-enigma/actions" rel="nofollow noopener">repository</a> and this <a href="https://github.com/linuxjuggler/cautious-octo-computing-machine" rel="nofollow noopener">repository</a> both have the workflow, which I copied from the <a href="https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file" rel="nofollow noopener">documentation.</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/ac3622b25212d2f5a7937d26e6d2f82f7b10c3bb.png" data-download-href="/uploads/short-url/ozs6NVxi0IsKoktCpiZZA5DsD6X.png?dl=1" title="Screen Shot 2019-09-02 at 9.20.43 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/27624/181095507-b684535b-ae84-4dde-8373-c03e63264637.png" alt="Screen Shot 2019-09-02 at 9.20.43 PM.png" data-base62-sha1="ozs6NVxi0IsKoktCpiZZA5DsD6X" width="690" height="282" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/ac3622b25212d2f5a7937d26e6d2f82f7b10c3bb_2_690x282.png, https://aws1.discourse-cdn.com/github/optimized/2X/a/ac3622b25212d2f5a7937d26e6d2f82f7b10c3bb_2_1035x423.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/a/ac3622b25212d2f5a7937d26e6d2f82f7b10c3bb_2_1380x564.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/ac3622b25212d2f5a7937d26e6d2f82f7b10c3bb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-02 at 9.20.43 PM.png</span><span class="informations">1886×771 50.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>And yes I have v2 enabled for my account</p>
<p><img src="https://user-images.githubusercontent.com/27624/181095508-b8b0fff4-4b52-4212-a658-643e4031e6d7.png" alt="Screen Shot 2019-09-02 at 9.30.52 PM.png" data-base62-sha1="g5cJ4MV01gBMx1uQSEfKHHE1xvP" width="593" height="213">But, if I was upgrading from v1 to v2, everything will work normally</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/44416e448addf7f5fe63839201197007025fc836.png" data-download-href="/uploads/short-url/9JOEsWmFXhGlob9mLjZ3q4ZJBfo.png?dl=1" title="Screen Shot 2019-09-02 at 9.32.11 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/27624/181095509-37ec211a-b5c6-48bf-8a27-105f848660c2.png" alt="Screen Shot 2019-09-02 at 9.32.11 PM.png" data-base62-sha1="9JOEsWmFXhGlob9mLjZ3q4ZJBfo" width="690" height="346" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/44416e448addf7f5fe63839201197007025fc836_2_690x346.png, https://aws1.discourse-cdn.com/github/original/2X/4/44416e448addf7f5fe63839201197007025fc836.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/44416e448addf7f5fe63839201197007025fc836.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/44416e448addf7f5fe63839201197007025fc836_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-02 at 9.32.11 PM.png</span><span class="informations">741×372 38.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So adding the same example workflow will work</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/872d448aca1a51a1ff734be9c3350b3b9259c72f.png" data-download-href="/uploads/short-url/jhPrxvOOGN0GZcpsYGMH3fnIwCb.png?dl=1" title="Screen Shot 2019-09-02 at 9.34.50 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/27624/181095510-dcd58fdc-2253-47a8-9305-ffbe6b0c7f0c.png" alt="Screen Shot 2019-09-02 at 9.34.50 PM.png" data-base62-sha1="jhPrxvOOGN0GZcpsYGMH3fnIwCb" width="690" height="346" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/872d448aca1a51a1ff734be9c3350b3b9259c72f_2_690x346.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/872d448aca1a51a1ff734be9c3350b3b9259c72f_2_1035x519.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/872d448aca1a51a1ff734be9c3350b3b9259c72f_2_1380x692.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/872d448aca1a51a1ff734be9c3350b3b9259c72f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-02 at 9.34.50 PM.png</span><span class="informations">1488×748 93.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>any idea why?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi <a class="mention" href="/linuxjuggler">@linuxjuggler</a> !</p>
<p>We’re fixing this bug. Thanks for sharing. If you’d like, I can press a button to upgrade all repositories in your account to the new syntax, so you don’t need to do it one by one. Let me know :).</p>

**Comment by zaherg** [comment]
<p>Hey <a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> thanks for the reply.</p>
<p>Upgrading all my existing repositories will be excellent.</p>
<p>Waiting for the fix :+1:</p>

**Comment by mscoutermarsh** [comment]
<p>Done! All repos in your account are now upgraded.</p>

**Comment by zaherg** [comment]
<p>Amazing, thanks !</p>
<p>sorry to ask but is there any ETA for the fix? or from now on any new repo I create will automatically get the new workflow?</p>

**Comment by zaherg** [comment]
<p>thanks <a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> I just test it with a new repo and it worked like a charm</p>
<p>thanks again :+1:</p>
